api/db-cdr/generate-cdr/libproject/genomic_manifests.rb

```diff
@@ -32,10 +38,12 @@
 # See the CDR playbook for more context on the overall publishing process:
 # https://docs.google.com/document/d/1St6pG_EUFB9oRQUQaOSO7a9UPxPkQ5n4qAVyKF9j9tk/edit#heading=h.xt7avgt1nsoh
 
-
 CURATION_PROD_SOURCE_CONFIG = {
   # Optional: to speed up iteration with this script, download and run against a local directory instead.
-  :wgs_aw4_prefix => "gs://prod-drc-broad/AW4_wgs_manifest/AoU_DRCB_SEQ_"
+  :wgs_aw4_prefix => "gs://prod-drc-broad/AW4_wgs_manifest/AoU_DRCB_SEQ_",
+  # The array_old_egt_files should be removed, likely in late 2022. This contains older
```

Comment:
There is a data reprocessing effort underway, which is not being released yet this time around, but will be in a future CDR release; this will be dependent on a call from the genomic curation team / stakeholders.